ios/LayoutReanimation/REASnapshot.m

```diff
@@ -79,6 +79,17 @@ - (void)makeSnapshotForView:(UIView *)view useAbsolutePositionOnly:(BOOL)useAbso
       // Identity matrix is an default value
       _values[@"transformMatrix"] = @[ @(1), @(0), @(0), @(0), @(1), @(0), @(0), @(0), @(1) ];
     }
+#if defined(RCT_NEW_ARCH_ENABLED) || TARGET_OS_TV
+    _values[@"borderRadius"] = @(0);
+#else
+    RCTView *reactView = (RCTView *)view;
```

Comment:
I think we need, because the `borderRadius` property comes from `RCTView` not from `UIView`.
I changed it a bit in https://github.com/software-mansion/react-native-reanimated/pull/4103/commits/8bf37c19b44d9999cc78494b228d6c5405ec9b97